tests/console-backend-service/internal/domain/eventing/trigger_test.go

```diff
@@ -0,0 +1,249 @@
+// +build acceptance
+
+package eventing
+
+import (
+	"fmt"
+	"testing"
+
+	"github.com/kyma-project/kyma/tests/console-backend-service/internal/resource"
+	v1 "k8s.io/api/core/v1"
+	"k8s.io/apimachinery/pkg/util/intstr"
+	"k8s.io/client-go/dynamic"
+
+	tester "github.com/kyma-project/kyma/tests/console-backend-service"
+	"github.com/kyma-project/kyma/tests/console-backend-service/internal/client"
+	"github.com/kyma-project/kyma/tests/console-backend-service/internal/graphql"
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+)
+
+func TestTriggerEventQueries(t *testing.T) {
+	c, err := graphql.New()
+	assert.NoError(t, err)
+
+	eventingCli, _, err := client.NewDynamicClientWithConfig()
+	require.NoError(t, err)
+
+	namespace, err := fixNamespace(eventingCli, t.Logf)
+	require.NoError(t, err)
+
+	_, err = fixService(eventingCli, t.Logf)
+	require.NoError(t, err)
+
+	subscription := subscribeTriggerEvent(c, createTriggerEventArguments(), triggerEventDetailsFields())
+	defer subscription.Close()
+
+	err = mutationTrigger(c, "create", createTriggerArguments(), triggerDetailsFields())
+	assert.NoError(t, err)
+
+	event, err := readTriggerEvent(subscription)
+	assert.NoError(t, err)
+
+	expectedEvent := newTriggerEvent("ADD", fixTrigger())
+	checkTriggerEvent(t, expectedEvent, event)
+
+	triggers, err := listTriggers(c, listTriggersArguments(), triggerDetailsFields())
+	assert.NoError(t, err)
+
+	checkTriggerList(t, triggers, 1, TriggerName)
+
+	err = mutationTrigger(c, "delete", deleteTriggerArguments(), metadataDetailsFields())
+	assert.NoError(t, err)
+
+	err = namespace.Delete(Namespace)
+	assert.NoError(t, err)
+}
+
+func newTriggerEvent(eventType string, trigger Trigger) TriggerEvent {
+	return TriggerEvent{
+		Type:    eventType,
+		Trigger: trigger,
+	}
+}
+
+func checkTriggerList(t *testing.T, query TriggerListQueryResponse, expectedLen int, expectedName string) {
+	assert.Equal(t, expectedLen, len(query.Triggers))
+	assert.Equal(t, expectedName, query.Triggers[0].Name)
+}
+
+func checkTriggerEvent(t *testing.T, expected, actual TriggerEvent) {
+	assert.Equal(t, expected.Type, actual.Type)
+	assert.Equal(t, expected.Trigger.Name, actual.Trigger.Name)
+	assert.Equal(t, expected.Trigger.Namespace, actual.Trigger.Namespace)
+}
+
+func readTriggerEvent(sub *graphql.Subscription) (TriggerEvent, error) {
+	type Response struct {
+		TriggerEvent TriggerEvent
+	}
+
+	var response Response
+	err := sub.Next(&response, tester.DefaultDeletionTimeout)
+
+	return response.TriggerEvent, err
+}
+
+func listTriggers(client *graphql.Client, arguments, resourceDetailsQuery string) (TriggerListQueryResponse, error) {
+	query := fmt.Sprintf(`
+		query{
+			triggers (
+				%s
+			){
+				%s
+			}
+		}
+	`, arguments, resourceDetailsQuery)
+	req := graphql.NewRequest(query)
+	var res TriggerListQueryResponse
+	err := client.Do(req, &res)
+
+	return res, err
+}
+
+func listTriggersArguments() string {
```

Comment:
You can test list query with subscriber parameter. Please think about it and extend test.